### DISCUSSION TITLE
How to hide "show" button in resource index page and still keep the record accessible to the user?

### DISCUSSION THREAD

**Discussion by hassyyy** [avo-hq/avo#858] [Help] [answered]
I have enabled `link_to_resource` for name field in my resource. Hence, the show button("eye" icon) in the index page is redundant. Is there a way to remove it?

I guess authorization won't be the optimal solution as I still want the user to be able to view the record

**Comment by adrianthedev** [comment, marked as answer]
Hey @hassyyy. Unfortunately no. They go hand in hand.

**Reply by hassyyy** [reply]
Alright :(
